packages/types/src/session.ts

```diff
@@ -169,6 +170,10 @@ export interface PublicUserData {
   userId?: string;
 }
 
+export interface SessionTasks {
+  orgs: Record<string, unknown>;
```

Comment:
For now, FAPI is returning an empty object by purpose but it might include some additional metadata about the task in the future so I'll update it by then 

The purpose of including it now is to unblock next PRs